packages/components/psammead-episode-list/src/index.jsx

```diff
@@ -69,18 +61,22 @@ EpisodeList.propTypes = {
   script: shape(scriptPropType).isRequired,
   service: string.isRequired,
   dir: oneOf(['ltr', 'rtl']),
+  darkMode: bool,
 };
 
 EpisodeList.defaultProps = {
   children: [],
   dir: 'ltr',
+  darkMode: false,
 };
 
 // This module also has a range of supplemental components to provide consumers with some compositational control
```

Comment:
I'm not sure what this sentence means. Also has a spelling mistake `compositational`. Is it basically saying this component is a compound component?

Reply:
Removed comment - seems unnecessary given the examples in the readme